File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var sumOfUnique = function(nums) {
    let output = {}
    for(let i=0;i<nums.length;i++){
        let ch = nums[i]
        output[ch] = (output[ch]||0)+1
    }
    let n =[]
    for(let key in output){
        if(output[key]===1){
            n.push(Number(key))
        }
    } 
    let sum =0
    for(let i=0;i<n.length;i++){
        sum= sum+n[i]
    }
    return sum
    
                
};

File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.py
class Solution(object):
    def sumOfUnique(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        f = {}
        output = 0
        for i in range(0,len(nums)):
            ch = nums[i]
            f[ch] = f.get(ch,0)+1
        for key in f:
            if f[key] == 1:
                output = output + int(key)
        return output